datalad/distributed/create_sibling_ria.py

```diff
@@ -191,9 +191,11 @@ class CreateSiblingRia(Interface):
             With `recursive=True`, only the top dataset will be aliased.""",
             constraints=EnsureStr() | EnsureNone()),
         post_update_hook=Parameter(
-            args=("--post-update-hook",),
-            doc="""Enable git's default post-update-hook for the created
-            sibling.""",
+            args=("--post-update-hook", "--update-server-info"),
```

Comment:
is adding an alternative --long-option really called for?  IMHO -- no need, and moreover we do not even guarantee that update-server-info is ran within the hook, we just do it once while running this command.

If insisting on keeping -- please adjust the title which might be used for the CHANGELOG entry to reflect that addition in the next .patch release.

Otherwise:
```suggestion
            args=("--post-update-hook",),
```

Reply:
I don't care much. `--post-update-hook` also does not convey that it *enables* the hook. I don't know what the title needs to be adjusted to, I am fine with any line.

Reply:
I could then argue that neither original nor added option reflect entirely the action they are intended to accomplish.  As such I do not see a point of having one more suboptimally named option to be added. I will leave it to someone else to finish review/merge this PR then.